ReactiveCocoa/AppKit/NSButton.swift

```diff
@@ -0,0 +1,29 @@
+import ReactiveSwift
+import enum Result.NoError
+import AppKit
+
+extension Reactive where Base: NSButton {
+
+	internal var associatedAction: Atomic<(action: CocoaAction<Base>, disposable: Disposable?)?> {
```

Comment:
Why is `disposable` optional? Seems like just the tuple needs to be optional? Not the `Disposable` inside?